tsconfig.types.json

```diff
@@ -47,15 +47,12 @@
 		],
 		"vscode/*": [
 			"./vscode/*"
-		],
-		"monaco-editor/esm/*": [
-			"./vscode/src/*"
 		]
 	},
 	"declaration": true,
 	"emitDeclarationOnly": true,
 	"skipLibCheck": true,
-	"types": ["monaco-editor", "node"]
+	"types": ["node"]
```

Comment:
I learnt this quite recently. Are you aware that this prevents the usage of any types defined in `node_modules/types`. It overrides it hard (see: https://www.typescriptlang.org/tsconfig#types). If you do this on purpose nevermind my comment.

Reply:
I... don't know, I don't think it was done on purpose.. I'll remove it after I check it causes no trouble

Reply:
FYI/warning; This can have effects/compilation problems as suddenly more types are visible for `tsc`

Reply:
I've check the build of the main project and it's fine, but I should have check the build of the demo, hold on...

Reply:
1a005f79c83032db5b628922640649fa3526a1cc

Reply:
👍